CRM/Utils/System/WordPress.php

```diff
@@ -339,20 +339,30 @@ public function url(
     $base = $this->getBaseUrl($absolute, $frontend, $forceBackend);
 
     // Overwrite base URL if we already have a front-end URL.
-    if (!$forceBackend && $script != '') {
-      $base = $script;
+    if (!$forceBackend && $frontend_url != '') {
+      $base = $frontend_url;
     }
 
     $queryParts = [];
     $admin_request = ((is_admin() && !$frontend) || $forceBackend);
 
+    /**
+     * Filter the Base URL.
+     *
+     * @since 5.66
```

Comment:
If we'd merged this yesterday this would be correct. Now it needs to be updated to 
```suggestion
     * @since 5.67
```